docs/monitoring-aws-lambda.asciidoc

```diff
@@ -28,6 +28,30 @@ image:images/architecture-white.png[image showing data flow from lambda function
 
 By using an AWS Lambda extension, Elastic APM agents can send data to a local Lambda extension process, and that process will forward data on to APM Server asynchronously. The Lambda extension ensures that any potential latency between the Lambda function and the APM Server instance will not cause latency in the request flow of the Lambda function itself.
 
+[[aws-lambda-overhead]]
+== Performance impact and overhead
+As described in the section on <<aws-lambda-arch>>, using Elastic APM with AWS Lambda requires to include the Elastic APM AWS Lambda extension and a corresponding Elastic APM agent to the Lambda runtime.
+These components may introduce a small overhead on the size of your function's deployment package as well as the execution duration of your function's invocations. 
+
+=== Impact on the deployment package size
+These compotents contribute a little to the uncompressed deployment package size of your Lambda function. The Elastic APM AWS Lambda extension and each of the APM agents have an uncompressed package size that is below 10MB. Thus, the overall impact of using Elastic APM on the uncompressed deployment package size of your Lambda function less than 20MB.
```

Comment:
```suggestion
These components contribute a little to the uncompressed deployment package size of your Lambda function. The Elastic APM AWS Lambda extension and each of the APM agents have an uncompressed package size that is below 10MB. Thus, the overall impact of using Elastic APM on the uncompressed deployment package size of your Lambda function is less than 20MB.
```

FWIW the node.js APM agent layer is *17MB* uncompressed:

```
% ./dev-utils/make-distribution.sh
...
Created build/dist/elastic-apm-node-lambda-layer.zip
...
% du -sh build/dist/nodejs
 17M	build/dist/nodejs
```

A dev build of the extension is 10MB for me.

Reply:
Note that if we made it a goal, I'm confident we could get that down under ~10MB pretty easily.